wai-aria/role/basic.html

```diff
@@ -7,17 +7,30 @@
 
 <div id='d' style='height: 100px; width: 100px' role="group" aria-label="test label"></div>
 <h1 id="h">test heading</h1>
+<p>Verifies <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
+<p role="foo" id="p">this is a paragraph</p>
+<span role="foo group" id="span">this is a span</span>
```

Comment:
I would prefer to keep basic very very simple. There are other issues for fallback and invalid role tests that you could self assign and create new files IMO: invalid_roles.html, fallback_roles.html, etc.

- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/11
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/19


I prefer to keep basic.html in a state where there are never failing tests in any implementation... Hence the reason to not rely on aria-utils.js in case some one breaks that with a future commit.

Reply:
If you use this PR for those, associate them as closing the 1–3 issues.

Reply:
Or just mention they are related to issues 11 and 19, since these 2 tests would not fully complete the work required for those.